### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by mrog**
*Issue #, if available:* CTECH-135

*Description of changes:* Minor code cleanup, including use of constants and fixing typos

*Testing performed:* Code built

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: api/v1beta1/cloudstackcluster_webhook_test.go [hunk 1]
```diff
@@ -20,30 +20,39 @@ import (
 	"context"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = Describe("CloudStackCluster webhooks", func() {
+	const (
```
Comment (rejoshed) [inline api/v1beta1/cloudstackcluster_webhook_test.go:27]: I'm okay with it, but I really don't think consts are necessary like this in test files.

File: pkg/cloud/instance.go [hunk 2]
```diff
@@ -163,9 +163,9 @@ func (c *client) GetOrCreateVMInstance(
 	setIfNotEmpty(csCluster.Spec.Account, p.SetAccount)
 	setIfNotEmpty(csCluster.Status.DomainID, p.SetDomainid)
 
-	// If this VM instance is a control plane, consider setting it's IP.
```
Comment (rejoshed) [inline pkg/cloud/instance.go:166]: Nice.

**Comment by rejoshed** [comment]
TBH, not sure I like the consts in the testing files, but not sure I care that much.

**Review by rejoshed — APPROVED** [review]

File: api/v1beta1/cloudstackcluster_webhook_test.go (hunk 1 above)
Comment (mrog) [reply api/v1beta1/cloudstackcluster_webhook_test.go:27]: They'd certainly be more important in production code, but I've experienced a fair number of test failures (or even worse, tests that don't fail when they should) due to mismatched literals.  If the same literal is used more than once, or is likely to be used again in the future, I like to use a constant or enum.

File: pkg/cloud/network.go [hunk 3]
```diff
@@ -31,6 +31,7 @@ const (
 	K8sDefaultAPIPort   = 6443
 	NetworkTypeIsolated = "Isolated"
 	NetworkTypeShared   = "Shared"
+	tcp                 = "tcp"
```
Comment (wongni) [inline pkg/cloud/network.go:34]:
```suggestion
	NetworkProtocolTCP                 = "tcp"
```
To be consistent with other constants.

Comment (mrog) [reply pkg/cloud/network.go:34]: Good idea.  Thanks!

**Pull request merged**